storage/eigen/include/algebra/storage/eigen.hpp

```diff
@@ -16,11 +16,11 @@
 namespace algebra::eigen {
 
 /// Array type used in the Eigen storage model
-template <typename T, std::size_t N>
+template <typename T, int N>
```

Comment:
Huh, apparently they actually use ```int``` in Eigen...

Reply:
Right.. with `std::size_t`, compiler ( I don't remember which one) gets error